core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/files/role-binding.yml

```diff
@@ -1,12 +1,38 @@
 apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
There are > 1 bindings, so I would suggest to rename this file to: `role-bindings.yml`